src/main/tests/dao/PCAPParserDaoImplTest.java

```diff
@@ -0,0 +1,109 @@
+package dao;
+
+import static org.junit.Assert.assertEquals;
+import com.google.netpcapanalysis.dao.PCAPParserDaoImpl;
+import com.google.netpcapanalysis.interfaces.dao.PCAPParserDao;
+import com.google.netpcapanalysis.models.PCAPdata;
+import com.google.netpcapanalysis.dao.PCAPDaoImpl;
+import com.google.netpcapanalysis.interfaces.dao.PCAPDao;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*; 
+import java.io.*;
+import org.apache.commons.io.IOUtils;
+import java.nio.charset.StandardCharsets;
+
+/* Compares file2.pcap's parsed results to hard-coded results. IP1 and IP2 addresses are located in hidden text file and retrieved as a stream. */
+public class PCAPParserDaoImplTest {
+  public PCAPParserDao parser;
+  public String PCAPNAME = "files/file2.pcap";
```

Comment:
Are these files generated or public ones.

Reply:
These are public ones, but the actual files are hidden (and accessed through class loader).